templates/single-product/add-to-cart/grouped.php

```diff
@@ -36,6 +36,7 @@
 				),
 				$product
 			);
+			$show_add_to_cart_button = false;
```

Comment:
should we add this variable to `woocommerce_grouped_product_list_before` and ` woocommerce_grouped_product_list_after` filters as well, looks like we are passing other variables that control if add to cart button should be displayed or not

Reply:
Thanks for pointing that out. Yes I have considered it but I think for now we can leave it out because those hooks are for the block of information on top and the "add to cart" button is outside of that group.

In addition because adding argument to a hook that already existed means it would be hard to remove in the future, I rather wait and see if anyone is actually needing that information before we decide to add it.